test/src/bumper.spec.js

```diff
@@ -102,39 +102,37 @@ describe('Bumper', () => {
                     eventManager.listenOnce(player, player.Event.AD_COMPLETED, () => {
                       eventManager.listenOnce(player, player.Event.AD_BREAK_END, () => {
                         eventManager.listenOnce(player, player.Event.PLAYING, () => {
-                          eventManager.listenOnce(player, player.Event.ENDED, () => {
```

Comment:
why it removed?

Reply:
post-bumper called on onPlaybackEnd(ended) and handled before so this ended called after and failed the test

Reply:
ENDED means the playback itself ended it must be triggered before AD_BREAK_START

Reply:
it's triggered before but our events are synchronized so it first handled in onPlaybackEnded and call immediately to play and ad_break_start so the ended which handled here will be after

Reply:
ended happened before but his previous handler calling to play so ad_break_start could call in the between ended handlers